tests/integration_tests/reports/alert_tests.py

```diff
@@ -61,43 +62,40 @@ def test_execute_query_as_report_executor(
     config: list[ExecutorType],
     expected_result: Union[tuple[ExecutorType, str], Exception],
     mocker: MockFixture,
-    app_context: None,
+    app_context: AppContext,
     get_user,
 ) -> None:
     from superset.commands.report.alert import AlertCommand
     from superset.reports.models import ReportSchedule
 
-    with app.app_context():
```

Comment:
The pushed app context is already being leveraged via the `app_context` fixture specified on line 65.